docs/maui/index.md

```diff
@@ -26,8 +26,9 @@ The .NET MAUI Community Toolkit supports the platforms officially supported by [
 * macOS 10.15 or higher, using Mac Catalyst.
 * Windows 11 and Windows 10 version 1809 or higher, using [Windows UI Library (WinUI) 3](/windows/apps/winui/winui3/).
 
-> [!NOTE]
-> While there is support for Tizen (provided by Samsung) in .NET MAUI, the .NET MAUI Community Toolkit does not currently support it.
+## Additional platform support
+
+.NET MAUI Community Toolkit also includes Tizen support, which is provided by Samsung.
```

Comment:
Should we include this in the section above (Supported versions). Of course that leads me onto the next question, do we have minimum version support for platforms through Tizen?

Reply:
I agree with @bijington. Let's keep all platforms together.
Minimum version is 6.5 according to csproj

Reply:
Yes, I just tried to align with what is written on [MAUI document](https://learn.microsoft.com/en-us/dotnet/maui/supported-platforms#additional-platform-support), but I think there is no reason to separate on here.

Reply:
I will include Tizen in the section above! :) Thanks for your suggestion.

Reply:
@VladislavAntonyuk That's true! The tricky part is..  Tizen 7.0 public which will be installed on the products has .net6 in it so basically to public developers 7.0 is the official version that supports .net6.